test.c

```diff
@@ -3742,6 +3742,29 @@ test_get_tree_provided ()
     CU_ASSERT (assert_apteryx_empty ());
 }
 
+void
+test_get_tree_provided_after_db ()
+{
+    const char *path = TEST_PATH"/interfaces/eth0/state";
+    GNode *root = NULL;
+    GNode *node = NULL;
+
+    CU_ASSERT (apteryx_provide (path, test_provide_cb));
+    CU_ASSERT (apteryx_set (path, "up"));
```

Comment:
Need to set this value to something other than "up" as that is the test_provide_cb answer.

Reply:
I think